firmware/ci.c

```diff
@@ -647,6 +647,10 @@ static void video_matrix_list(void)
 #ifdef CSR_HDMI_IN1_BASE
 	wprintf("input1 (1): %s\n", HDMI_IN1_MNEMONIC);
 	wputs(HDMI_IN1_DESCRIPTION);
+#endif
+#ifdef CSR_PCIE_PHY_BASE
+	wprintf("pcie:\n");
```

Comment:
Should probably add a short code for PCIe?

Reply:
`p` is used by pattern so maybe `e`?

Reply:
I've added `e` as a short code for PCIe input